src/v/redpanda/application.cc

```diff
@@ -126,6 +131,14 @@ void application::initialize(std::optional<scheduling_groups> groups) {
     _scheduling_groups.create_groups().get();
     _deferred.emplace_back(
       [this] { _scheduling_groups.destroy_groups().get(); });
+
+    if (proxy_client_cfg) {
+        _proxy_client_config.read_yaml(*proxy_client_cfg);
```

Comment:
can you remind me why the configuration for the proxy client is not taken from redpanda.yml via something like

```
proxy_client_xxx
copro_client_xxx
```

which would be the alternative iiuc?

Reply:
nevermind. I see now from the example in the cover letter how this works.